src/code/z_collision_check.c

```diff
@@ -1408,6 +1408,7 @@ void CollisionCheck_NoBlood(PlayState* play, Collider* collider, Vec3f* v) {
  * Used by collider types HIT0 and HIT8.
  */
 void CollisionCheck_BlueBlood(PlayState* play, Collider* collider, Vec3f* v) {
+    static s32 pad;
```

Comment:
Why `sBssDummyNeg1` instead of renumbering? Keeping it consistent helps with debugging bss ordering, it's surprising that `sBssDummy0` is not in fact the first dummy variable

Reply:
because lazy and preserve diff and blame (especially for something as sketchy as this)

there may even be a better solution to this bss stuff now that it seemingly changed (no clue), but it's complete wizardry to me

I'll renumber them from 0